frontends/android/BitBoxApp/app/src/main/java/ch/shiftcrypto/bitboxapp/MainActivity.java

```diff
@@ -595,6 +666,7 @@ protected void onDestroy() {
             unbindService(connection);
         }
         super.onDestroy();
+        connectivityManager.unregisterNetworkCallback(networkCallback);
```

Comment:
`if (connectivityManager != null && networkCallback != null) {` for robustness.

I also just noticed that the register/deregister pair does not match. onCreate pairs with onDestroy, and onStart pairs with onStop. but currently the stuff is registered in onStart. 

Would it make sense to just move the setup to onCreate instead?